ext/scheduler/airflow/dag/compiler.go

```diff
@@ -61,28 +57,31 @@ func (c *Compiler) Compile(jobDetails *scheduler.JobWithDetails) ([]byte, error)
 		Upstreams:       upstreams,
 	}
 
+	airflowVersion, err := project.GetConfig(tenant.ProjectSchedulerVersion)
+	if err != nil {
+		msg := fmt.Sprintf("%s is not provided in project %s, %s", tenant.ProjectSchedulerVersion, project.Name(), err.Error())
```

Comment:
This will return an error in case the configuration is not defined, we want to fail in that case or have a fallback version here ?

Reply:
I think it's better to provide a mechanism to use the fallback version and logged as a warning